src/Core/SettingsChangesHistory.h

```diff
@@ -115,6 +115,7 @@ static const std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges
               {"output_format_csv_serialize_tuple_into_separate_columns", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_deserialize_separate_columns_into_tuple", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_try_infer_strings_from_quoted_tuples", true, true, "A new way of how interpret tuples in CSV format was added."},
+              {"lightweight_mutation_projection_mode", "throw", "throw", "When lightweight delete happens on a table with projection(s), the possible operations include throw the exception as projection exists, or drop all projection related to this table then do lightweight delete."},
```

Comment:
The previous value should be `ignore` for compatibility.

Reply:
I think should be throw here, because even before this pr and another [pr](https://github.com/ClickHouse/ClickHouse/pull/64212), it is throw already. Though for other types of mergetree, ignore would be the default, but regular mergetree would be the majority I think.

Reply:
discussed with Nikolai, should throw then.